install_on_arm_board.md

```diff
@@ -38,4 +38,4 @@
 
 ---
 
-***If you need help during one of these steps, do not hesitate to use [our support tools](/support).***
+***If you need help during one of these steps, do not hesitate to use [our support channel](/support).***
```

Comment:
It was meant as channel in the sense of "canal" and not as "chat". I agree this can be confusing. What do you think of "You can get support if you need help during one of these steps" or "Need help during one of these steps? Get support!"